### ISSUE TITLE
password_file option missing from /opt/mosquitto.conf

### ISSUE THREAD

**Issue by gbrayut** [hassio-addons/addon-mqtt#17]
# Problem/Motivation

Currently the conf file has an acl_file but is missing the password_file entry. This means you can login without a password as long as you specify a valid username.

## Expected behavior

It should validate the password for user accounts using the /opt/pwfile file

## Actual behavior

It will accept any password so long as you specify a valid account.

## Steps to reproduce

Test using the following configuration
```
{
  "log_level": "info",
  "certfile": "fullchain.pem",
  "keyfile": "privkey.pem",
  "web": {
    "enabled": true,
    "ssl": true
  },
  "broker": {
    "enabled": true,
    "enable_ws": false,
    "enable_mqtt": true,
    "enable_ws_ssl": true,
    "enable_mqtt_ssl": true,
    "allow_anonymous": false
  },
  "mqttusers": [
    {
      "username": "ro",
      "password": "doesnotmatter",
      "readonly": true,
      "topics": [
        "#"
      ]
    },
    {
      "username": "hass",
      "password": "doesnotmatter",
      "readonly": false,
      "topics": [
        "#"
      ]
    },
    {
      "username": "iot",
      "password": "doesnotmatter",
      "readonly": false,
      "topics": [
        "#"
      ]
    }
  ]
}
```
## Proposed changes

The following should be appended to the /opt/mosquitto.conf file

password_file /opt/pwfile

**Comment by ludeeus** [comment]
Thanks for reporting this @gbrayut

**Comment by ludeeus** [comment]
This is corrected in 0.3.1 that just got released.
<https://github.com/hassio-addons/addon-mqtt/releases/tag/v0.3.1>
